pkg/identity/identity.go

```diff
@@ -268,6 +268,9 @@ func LookupReservedIdentityByLabels(lbls labels.Labels) *Identity {
 		if option.Config.PolicyCIDRMatchesNodes() {
 			return nil
 		}
+		if option.Config.PerNodeIdentitiesEnabled() {
```

Comment:
Can you add a comment here that explains why we are returning here (similar to `if` clause above this code)?

Reply:
done